.github/actions/publish_to_pages_production/action.yml

```diff
@@ -0,0 +1,12 @@
+name: publish_to_pages_production
+description: Publish to cloudflare pages (production)
+runs:
+  using: composite
+  steps:
+  - name: Publish to cloudflare pages (production)
+    run: |-
+      npm i wrangler@3.1.0
+      cd packages/core
+      npx wrangler pages deploy dist/ --project-name=deriv-app-pages --branch=main
```

Comment:
I can also combine this action with the other one for staging, for now and for better clearance I've written them separated.